scripts/xdist/get_worker_test_list.py

```diff
@@ -0,0 +1,47 @@
+"""
+This script strips the console log of a pytest-xdist Jenkins run into the test list
+of an individual worker.
+Assumes the following format:
+[test-suite] [worker] RESULT test
+"""
+import click
+import io
+
+
+@click.command()
+@click.option(
+    '--log-file',
+    help="File name of console log .txt file from a Jenkins build "
+    "that ran pytest-xdist.",
+    required=True
+)
+@click.option(
+    '--worker',
+    help="Pytest worker that ran the test list. Example: gw0",
+    required=True
+)
+@click.option(
+    '--test-suite',
+    help="Test suite that the pytest worker ran. Example: lms-unit",
+    required=True
+)
```

Comment:
I was unaware that you could feed pytest a list of tests to run. This seems pretty useful. I would suggest changing this script to create a file for each worker. It should be pretty quick to do so, and the files are small. This way, if we choose to incorporate this into Jenkins tooling, you will have a file for each worker, easy to download and run locally